app/models/manageiq/providers/autosde/inventory/collector/storage_manager.rb

```diff
@@ -4,7 +4,9 @@ class ManageIQ::Providers::Autosde::Inventory::Collector::StorageManager < Manag
   attr_accessor :manager
 
   def physical_storages
-    @physical_storages ||= @manager.autosde_client.StorageSystemApi.storage_systems_get.map do |system|
+    @physical_storages ||= @manager.autosde_client.StorageSystemApi.storage_systems_get.map
```

Comment:
`.map` without a block is a little strange, if you just want to turn an enumerator into an array you can use `.to_a`

Reply:
I moved all the parser code from the collector into the parser. this is not an issue anymore